examples/cndpfwd/main.c

```diff
@@ -147,6 +147,62 @@ _fwd_test(jcfg_lport_t *lport, struct fwd_info *fwd)
     return n_pkts;
 }
 
+static int
+_l3fwd_test(jcfg_lport_t *lport, struct fwd_info *fwd)
+{
+    struct fwd_port *pd                          = lport->priv_;
+    struct create_txbuff_thd_priv_t *thd_private = pd->thd->priv_;
+    txbuff_t **txbuff;
+    int i, n_pkts;
+    struct ether_addr eaddr;
+    uint16_t tx_port;
+
+    if (!pd)
+        CNE_ERR_RET("fwd_port passed in lport private data is NULL\n");
+
+    txbuff = thd_private->txbuffs;
+
+    n_pkts = __rx_burst(fwd->pkt_api, pd, pd->rx_mbufs, fwd->burst);
+    if (n_pkts == PKTDEV_ADMIN_STATE_DOWN)
+        return -1;
+
+    for (i = 0; i < n_pkts; i++) {
```

Comment:
Minor nit. On line 192 we use 'int i = 0', but here we use i = 0 with a global int i shadowing i. I would suggest to remove the global int i and change this one to int i = 0 to be consistent.